x/payment/types/payfordata_test.go

```diff
@@ -157,7 +157,7 @@ func TestCreateCommitment(t *testing.T) {
 			k:         4,
 			namespace: bytes.Repeat([]byte{0xFF}, 8),
 			message:   bytes.Repeat([]byte{0xFF}, 11*ShareSize),
-			expected:  []byte{0xf2, 0xd4, 0xfc, 0x39, 0x4e, 0xf3, 0x97, 0x9d, 0xf4, 0x4c, 0x99, 0x87, 0x36, 0x7d, 0x7d, 0x4, 0xf2, 0xa7, 0x89, 0x26, 0x6d, 0xf5, 0x78, 0xe1, 0xff, 0x72, 0xb4, 0x75, 0x12, 0x1e, 0x71, 0xc3},
+			expected:  []byte{0xe3, 0xaa, 0xdb, 0x1c, 0x2e, 0xc3, 0x77, 0x4a, 0x46, 0xb6, 0xb2, 0xf0, 0x4f, 0x6b, 0x3d, 0x4f, 0x17, 0x99, 0x82, 0x6b, 0xd6, 0xd4, 0x40, 0x5e, 0xbe, 0x1b, 0x1f, 0x29, 0x5a, 0x53, 0x3f, 0xbb},
```

Comment:
[no change needed] it's difficult to determine if this change is desirable but we already have a [todo](https://github.com/celestiaorg/celestia-app/blob/320ab5be792199b294845ee57afc01d84659b955/x/payment/types/payfordata_test.go#L146) to fix.

Reply:
we should expect the commitment to change since we are adding bytes the leafs and inner nodes. This test is basically just to make sure that we don't accidentally do that. So its desirable that something happened, but I obvi didn't manually calculate this myself, so I'm unsure if its 100% what we want.

Reply:
by manually calculate, I mean like, literally hand calculate the values. I did generate the commitment by hand double check the work.

Reply:
> I did generate the commitment by hand double check the work.

can you share how I would do this? If the expected bytes are indeed what we want, then approving to unblock

Reply:
yeah, we can be super lazy and just run this test since its creating the commitment for us, or we can write a quick script that does the same